package.json

```diff
@@ -55,13 +55,12 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@types/js-yaml": "^4.0.1",
-        "@types/node": "^10.12.0",
+        "@types/node": "^20.3.1",
```

Comment:
That's a huge bump.

Reply:
yeah, this is just the typings though so it doesn't really affect anything except compilation. Seems to be ok (aside from the fact that we'll no longer support node 14)

Reply:
fwiw, this is also what we have in the `main` branch, in `release-1.x` we didn't have dependabot turned on, so it didn't get updated for a long time, and then there were some compilation fixes that were needed.